common/cmac.c

```diff
@@ -80,12 +80,11 @@ bool cmac_calc(uint8_t *m, uint8_t bytes, uint8_t *data_prefix, bool check)
 		}
 		xtea_enc(buf, buf, K_mac);
 	}
-	if (check) {
+	if (check)
 		for (i = 0; i < 4; i++)
 			if (m[bytes + i] != buf[i]) return false;
-	} else {
-		memcpy(m + bytes, buf, 4);
-	}
+			else

```

Comment:
This looks like the beautifier changed the function of the code, I think the `else` was meant to the outer `if`? This is one of the cases where I think removing curly braces aggresivelly like this can hurt.

Reply:
yeah, that's definitely a bug, good you got it, thanks!